browser/resources/settings/BUILD.gn

```diff
@@ -65,8 +65,14 @@ generate_grd("build_grd") {
   deps = [ ":preprocess" ]
 
   if (!is_android) {
-    grdp_files = [ "$root_gen_dir/brave/web-ui-commands/commands.grdp" ]
-    deps += [ "shortcuts_page:commands_ui" ]
+    grdp_files = [
+      "$root_gen_dir/brave/web-ui-commands/commands.grdp",
+      "$root_gen_dir/brave/web-ui-email_aliases/email_aliases.grdp",
```

Comment:
this should not be behind `!is_android` because the rest of the feature is not guarded by `!is_android`. Also this doesn't seem like the right place for this since there is nothing else here for other settings ui? @petemill @fallaciousreasoning ?

Reply:
I think this is the right place - this UI has been written in React which is a bit different to most of the other settings pages (except `commands`/`shortcuts`). It makes sense for this UI to be in React, as its a fairly standalone page and we might end up showing it in a separate modal over a webpage.

Reply:
I see, but the `!is_android` flag definitely doesn't apply here

Reply:
moved it outside the !is_android flag